meta-iot2000-bsp/recipes-kernel/linux/linux-cip-rt_4.4.bb

```diff
@@ -4,12 +4,10 @@ SECTION = "kernel"
 require recipes-kernel/linux/linux-yocto.inc
 require linux-cip_4.4.inc
 
-LINUX_VERSION = "4.4.235-cip49-rt31"
```

Comment:
Not needed anymore?

Reply:
I think  I made some mistake. Has add this back and update to the newer CIP version